crates/re_viewer_context/src/annotations.rs

```diff
@@ -21,6 +21,13 @@ pub struct Annotations {
 }
 
 impl Annotations {
+    pub fn missing() -> Self {
```

Comment:
Make it #[inline] if it needs to be public (does it need to be public though, isn't the expected entrypoint the static var below?)

Reply:
Using it directly in the benchmarks seems a lot simpler than involving `MISSING_ANNOTATIONS` which is an `Arc`.